managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -1602,7 +1605,7 @@ synchronized void createLedgerAfterClosed() {
     @Override
     public void rollCurrentLedgerIfFull() {
         log.info("[{}] Start checking if current ledger is full", name);
-        if (currentLedgerEntries > 0 && currentLedgerIsFull()) {
+        if (currentLedger != null && currentLedgerEntries > 0 && currentLedgerIsFull()) {
```

Comment:
Will it make sense to exit early from this method if there is no 'currentLedger' ?

Reply:
essentially this is it.